C#HomeWork/thu thap quang/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿Console.WriteLine("Hay nhap so quang da dao duoc: ");   
+int totalOres = Convert.ToInt32(Console.ReadLine());
+int coins = 0;
+
+int ores = Math.Min(totalOres, 10);
```

Comment:
```suggestion
int ores = Math.Min(totalOres, 10);
```

cái số 10 nên dùng hằng số để thể hiện cho dễ hiểu hơn khi đọc code nè